src/eradiate/data/_store.py

```diff
@@ -13,7 +13,7 @@
 data_store: MultiDataStore = None
 
 
-def init_data_store(offline: bool | None = None) -> None:
+def init_data_store(offline: bool | None = None, production: bool | None = None) -> None:
```

Comment:
Docstring for this new option is missing. Side comment: the end goal is to generalize the data store setup with a config file-based system and provide defaults for dev and production setups, while still allowing the user to override it.